modules/ROOT/pages/sdk-xattr-example.adoc

```diff
@@ -6,7 +6,7 @@ The Java SDK supports creation and handling of _extended attributes_.
 == Subdocument Operations and Extended Attributes
 
 A high-level summary of extended attributes can be found in xref:sdk-xattr-overview.adoc[Extended Attributes].
-Extended attributes are handled by means of extensions to the _Subdocument API_.
+Extended attributes (XATTR) are handled by means of extensions to the _Subdocument API_.
```

Comment:
handled?  isn't accessed more appropriate?

Also, we decided that Sub-Document should be hyphenated and camel cased IIRC way back when.  The sdk-rfc covers this.  

Also, wouldn't the parenthesized bit be with an "s" since the thing it's talking about is plural?